llpc/context/llpcCompiler.cpp

```diff
@@ -728,6 +728,221 @@ template <typename T> static FormattedBytes formatBytesLittleEndian(ArrayRef<T>
   return format_bytes(bytes, /* FirstByteOffset = */ None, /* NumPerLine = */ 16, /* ByteGroupSize = */ 1);
 }
 
+// Build unlinked shader to ElfPackage with pipeline info.
+//
+// @param [in]  pipelineInfo : Info to build this shader module
+// @param [out] pipelineOut : Output of building this shader module
+// @param [in]  stage : Shader stage of needing to compile
+// @param pipelineDumpFile : Handle of pipeline dump file
+// @returns : Result::Success if successful. Other return codes indicate failure.
+Result Compiler::buildGraphicsShaderStage(const GraphicsPipelineBuildInfo *pipelineInfo,
+                                          GraphicsPipelineBuildOut *pipelineOut, UnlinkedShaderStage stage,
+                                          void *pipelineDumpFile) {
+  BinaryData elfBin = {};
```

Comment:
Can we define `elfBin` closer to where it's first used? This function is long and I cannot quickly find its uses outside of an IDE.